modules/shortcodes/instagram.php

```diff
@@ -139,40 +138,35 @@ function jetpack_instagram_handler( $matches, $atts, $url ) {
 	}
 
 	if ( ! empty( $response_body->html ) ) {
-		if ( ! $did_script ) {
-			$did_script = true;
-			add_action( 'wp_footer', 'jetpack_instagram_add_script' );
-		}
-
-		// there's a script in the response, which we strip on purpose since it's added above
+		wp_enqueue_script( 'jetpack-instagram-embed', plugins_url( 'js/instagram.js', __FILE__ ), array( 'jquery' ) );
```

Comment:
Please consider updating this enqueue to `wp_enqueue_script( 'jetpack-instagram-embed', plugins_url( 'js/instagram.js', __FILE__ ), array( 'jquery' ), 1.0, true );` to ensure that it is loaded at the end of the page. The script specifically relies on the `<body>` tag being present, so it makes sense to specify that it is loaded in the footer to prevent it from being loaded early.